apps/nuxt-ssr/pages/_schema/ssr-catalogue/cohorts/_cohort.vue

```diff
@@ -12,155 +12,167 @@
+                <p class="card-text">
+                  <small class="text-muted">
+                    <router-link
+                      :to="`/networks/${network.pid}`"
+                      class="stretched-link"
+                      >LEARN MORE
+                    </router-link>
+                  </small>
+                </p>
+              </template>
+            </image-card>
+          </div>
+        </grid-block>
 
-      <grid-block heading="Subpopulations" v-if="subpopulations.length">
-        <table-display
-          :isClickable="true"
-          @row-click="$router.push({ path: $event._path })"
-          :columns="[
-            { name: 'name', label: 'Name' },
-            { name: 'description', label: 'Description' },
-            { name: 'numberOfParticipants', label: 'Number of participants' },
-            { name: 'ageGroups', label: 'Age categories' },
-          ]"
-          :rows="subpopulations"
-        ></table-display>
-      </grid-block>
+        <grid-block heading="Available data & samples">
+          <strong>Data categories</strong>
+          <p>{{ dataCategories.join(", ") }}</p>
+          <strong>Sample categories</strong>
+          <p>{{ sampleCategories.join(", ") }}</p>
+          <strong>Areas of information</strong>
+          <p>{{ areasOfInformation.join(", ") }}</p>
+        </grid-block>
 
-      <grid-block heading="Collection events" v-if="collectionEvents.length">
-        <table-display
-          :isClickable="true"
-          @row-click="$router.push({ path: $event._path })"
-          :columns="[
-            { name: 'name', label: 'Name' },
-            { name: 'description', label: 'Description' },
-            { name: 'startAndEndYear', label: 'Start and end year' },
-          ]"
-          :rows="collectionEvents"
-        ></table-display>
-      </grid-block>
+        <grid-block heading="Documentation" v-if="documentation.length">
+          <links-list :items="documentation"></links-list>
+        </grid-block>
 
-      <div class="card-columns card-columns-2">
-        <grid-block heading="Access conditions">
-          <ul>
-            <li
-              v-for="(condition, index) in cohort.dataAccessConditions"
-              :key="index"
-            >
-              {{ condition.name }}
-            </li>
-          </ul>
-          <p>{{ cohort.dataAccessConditionsDescription }}</p>
-          <p>{{ cohort.releaseDescription }}</p>
+        <grid-block heading="Subpopulations" v-if="subpopulations.length">
+          <table-display
+            :isClickable="true"
+            @row-click="$router.push({ path: $event._path })"
+            :columns="[
+              { name: 'name', label: 'Name' },
+              { name: 'description', label: 'Description' },
+              { name: 'numberOfParticipants', label: 'Number of participants' },
+              { name: 'ageGroups', label: 'Age categories' },
+            ]"
```

Comment:
Would be nice to put a variable here (and some other places in this file) to keep the html a bit more readable.